.babel-preset.js

```diff
@@ -47,7 +47,7 @@ function preset(context, options = {}) {
         {
           // we are only polyfilling the node environment
           // so we need to enable the runtime replacements for the browser preset
-          polyfill: !!browser,
+          corejs: browser ? 2 : false,
```

Comment:
present env should handle this i believe since it's got the selective polyfilling on already?

Reply:
This might be the case but I was matching the previous behaviour - shall I remove this? And should I keep the changes in the last commit with the Babel runtime dependencies, or can these be left as just @babel/runtime?

Reply:
I'd say we want to remove it if it doesn't break anything. The reason they added this was exactly to enable less unnecessary polyfilling so it'd be a win to change, even if it's not the same as previous behavior

Reply:
Okay, I'll remove it and run through the tests. And shall I undo [this commit](https://github.com/gatsbyjs/gatsby/pull/7655/commits/f3c86df8f8d756bf899695ee5bde5f1d5830900e) too? (Tests were passing before I added this)